keyboards/kyria/keymaps/j-inc/rules.mk

```diff
@@ -4,4 +4,4 @@ RGBLIGHT_ENABLE = yes      # Enable keyboard RGB underglow
 WPM_ENABLE = yes
 MOUSEKEY_ENABLE = no
 BOOTMAGIC_ENABLE = no
-EXTRAFLAGS += -flto
+LTO_ENABLE = yes
```

Comment:
Actually, potential issue here: setting `LTO_ENABLE = yes` flips the keymap while keeping the OLEDs on the correct sides. Might want to set this to `no` until this issue is resolved.

`#define MASTER_RIGHT` is ignored with `LTO` enabled in this case. Plugging into the left half provides normal functionality.